repository/src/main/java/org/lfenergy/compas/scl/data/util/JaxbMarshaller.java

```diff
@@ -0,0 +1,27 @@
+// SPDX-FileCopyrightText: 2021 Alliander N.V.
+//
+// SPDX-License-Identifier: Apache-2.0
+package org.lfenergy.compas.scl.data.util;
+
+import org.lfenergy.compas.scl.data.model.Item;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Unmarshaller;
+import javax.xml.transform.stream.StreamSource;
+import java.io.StringReader;
+
+public class JaxbMarshaller {
+    private JAXBContext jaxbContext;
+    private Unmarshaller jaxbUnmarshaller;
+
+    public JaxbMarshaller() throws JAXBException {
+        jaxbContext = JAXBContext.newInstance("org.lfenergy.compas.scl.data.model");
```

Comment:
No, I prefer to only put property in a property file if they are configurable.
This will only change if the code of the model classes moves to a other location, so code change.